README.md

```diff
@@ -55,6 +62,11 @@ pip install torchserve torch-model-archiver
     source activate torchserve
     ```
 
+2. Optional if using torchtext models
+    ```bash
+    pip install sentencepiece
```

Comment:
I'd remove this and add instructions to the torchtext models folder.
There are many issues with this and it confuses the flow for basic setup.
For example, this doesn't work on mac w/ py3.8 - so does that get mentioned here? No, it's too specific.